pkgs/applications/graphics/flaca/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "flaca";
+  version = "2.2.1";
+
+  src = fetchFromGitHub {
+    owner = "Blobfolio";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-RXMqPpQM2h6EtIIH8Ix31euC7zK3v2QohZqouNlK7rM=";
+  };
+
+  cargoPatches = [ ./cargo-lock.patch ];
+  cargoHash = "sha256-z8nqpaSz5dZuoDJUgX1n+6gwFYsS42xwgCn7RnzBXbw=";
```

Please use the following instead :)

```suggestion
  # upstream does not provide a Cargo.lock
  cargoLock.lockFile = ./Cargo.lock;
  postPatch = ''
    ln -s ${./Cargo.lock} Cargo.lock
  '';
```

And replace `./cargo-lock.patch` with the actual `Cargo.lock` (plain, not as `diff`)

---

thank you for your suggestion!